pallet/src/api.rs

```diff
@@ -14,6 +14,8 @@ pub struct MoveApiEstimation {
     pub gas_used: u64,
     /// Status code for the MoveVM execution.
     pub vm_status_code: u64,
+    /// Substrate weight.
+    pub weight: Weight,
```

Comment:
I know it's a huge name but it tells the story 😄 
```suggestion
    /// Substrate weight required for the complete extrinsic cost combined with the variable gas indicated in the `Estimaton` struct.
    pub total_weight_cost_including_gas_used: Weight,
```